client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -97,24 +101,19 @@ const ConnectedSingleSiteJetpack = connectOptions( ( props ) => {
 				align="left"
 			/>
 			<CurrentTheme siteId={ siteId } />
-			{ ! shouldShowOfferResetFlow() &&
-				! requestingSitePlans &&
-				currentPlan &&
-				! hasUnlimitedPremiumThemes &&
-				! isPartnerPlan && (
-					<UpsellNudge
-						forceDisplay
-						plan={ PLAN_JETPACK_BUSINESS }
-						title={ translate( 'Access all our premium themes with our Professional plan!' ) }
-						description={ translate(
-							'In addition to our collection of premium themes, ' +
-								'get Elasticsearch-powered site search, real-time offsite backups, ' +
-								'and security scanning.'
-						) }
-						event="themes_plans_free_personal_premium"
-						showIcon={ true }
-					/>
-				) }
+			{ ! requestingSitePlans && currentPlan && ! hasUnlimitedPremiumThemes && ! isPartnerPlan && (
+				<UpsellNudge
+					forceDisplay
+					title={ translate( 'Get unlimited premium themes' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Unlimited premium themes', { context: 'logged-out-homepage'} )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/improve/premium-themes-upsell-copy)